glmmbernoulli.qmd

```diff
@@ -0,0 +1,523 @@
+  fit(MixedModel, form, contra, Bernoulli(); contrasts, nAGQ)
+end
+```
+
+Comparing this fitted model to the previous one
+
+```{julia}
+MixedModels.likelihoodratiotest(com03, com04)
+```
+
+confirms the usefulness of this term.
+
+A series of such model fits led to a model with random effects for the combinations of `dist` and `urban`, because differences between urban and rural women in the same district were comparable to differences between districts, even after accounting for an effect of `urban` at the fixed-effects (or population) level.
+
+```{julia}
+com05 = let
+  form = @formula(
+    use ~ 1 + urban + ch * age + age & age + (1 | dist & urban)
+  )
+  fit(MixedModel, form, contra, Bernoulli(); contrasts, nAGQ)
+end
+```
+
+In more detail,
+
+```{julia}
+#| code-fold: true
+println(com05)
+```
+
+Notice that, although there are 60 distinct districts, there are only 102 distinct combinations of `dist` and `urban` represented in the data.
+In 15 of the 60 districts there are no rural women in the sample and in 3 districts there are no urban women in the sample, as shown in a frequency table
+
+```{julia}
+#| code-fold: true
+freqtable(contra, :urban, :dist)
+```
+
+## Link functions and interpreting coefficients {#sec-glmmlink}
+
+To this point the only difference we have encountered between fitting generalized linear mixed models (GLMMs) and linear mixed models (LMMs) is the need to specify the distribution family in a call to `fit`.
+The formula specification is identical and the assessment of the significance of terms using likelihood ratio tests is similar.
+This is intentional.
+We have emphasized the use of likelihood ratio tests on terms, whether fixed-effects or random-effects terms, exactly so the approach will be general.
+
+However, the interpretation of the coefficient estimates in the different types of models is different.
+In a linear mixed model the conditional mean (or "expected value") of the response given the random effects is simply the value of the *linear predictor*, $\bbmu=\bbeta=\bbX\bbbeta+\bbZ\bbb$ .
+That is, if we assume that we know the values of the fixed-effects parameters, $\bbbeta$, and the random effects, $\bbb$, then the expected response for a particular combination of covariate values is a linear combination of these coefficients where the particular linear combination is determined by values of the covariates for that observation.
+Individual coefficients can be interpreted as slopes of the fitted response with respect to a numeric covariate or as shifts between levels of a categorical covariate.
+
+It is worthwhile emphasizing this relationship, and how it is used to form predictions from the model, by illustrating the process on a grid of covariate values.
+To this end, we will take a brief excursion into creating grids of covariate values and evaluating linear predictors.
+
+### Creating grids of covariate values {#sec-covariategrid}
```

Comment:
I think this is a good segue to link functions, but it might also be nice to point out in a footnote that Effects.jl was developed exactly for this type of plotting and that there's also a `predict(::GeneralizedLinearMixedModel)` method.